web/onos-gui/src/app/onos-topo/entity-list/entity-list.component.html

```diff
@@ -0,0 +1,82 @@
+            <input id="searchinput" [(ngModel)]="queryStr" type="search" #search placeholder="Search" />
+            <!-- <select [(ngModel)]="queryBy">
+                <option value="" disabled>Search By</option>
+                <option value="id">Id</option>
+                <option value="display">Display</option>
+                <option value="version">Version</option>
+                <option value="type">Type</option>
+                <option value="address">Address</option>
+                <option value="revision">Revision</option>
+                <option value="target">Target</option>
+                <option value="timeout">Timeout</option>
+            </select> -->
+        </div>
+    </div>
+
+    <div id="summary-list" class="summary-list">
+        <table class="table table-sticky">
+            <thead>
+                <tr>
+                    <th colId="id" (click)="onSort('id')">Id
+                        <onos-icon classes="active-sort" [iconSize]="10" [iconId]="sortIcon('id')"></onos-icon>
+                    </th>
+                    <th colId="displayName" (click)="onSort('displayName')">Display Name
+                        <onos-icon classes="active-sort" [iconSize]="10" [iconId]="sortIcon('displayName')">
+                        </onos-icon>
+                    </th>
+                    <th colId="version" (click)="onSort('version')">Version
+                        <onos-icon classes="active-sort" [iconSize]="10" [iconId]="sortIcon('version')">
+                        </onos-icon>
+                    </th>
+                    <th colId="type" (click)="onSort('type')">Type
+                        <onos-icon classes="active-sort" [iconSize]="10" [iconId]="sortIcon('type')"></onos-icon>
+                    </th>
+                    <th colId="entity" (click)="onSort('entity')">Entity
+                        <onos-icon classes="active-sort" [iconSize]="10" [iconId]="sortIcon('entity')"></onos-icon>
+                    </th>
+                    <th colId="kind" (click)="onSort('kind')">Kind
+                        <onos-icon classes="active-sort" [iconSize]="10" [iconId]="sortIcon('kind')"></onos-icon>
+                    </th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr *ngIf="topoDeviceService.entityList.size === 0" class="no-data">
+                    <td colspan="9">{{ annots.noRowsMsg }}</td>
+                </tr>
+                <tr *ngFor="let entity of topoDeviceService.entityList | keyvalue"
+                    (click)="selectCallback($event, entity.key, entity.value)">
+                    <td>{{ entity.value.getId() }}</td>
+                    <td>{{ printMapValue(entity.value.getAttributesMap(), 'displayname') }}</td>
+                    <td>{{ printMapValue(entity.value.getAttributesMap(), 'version') }}</td>
+                    <td>{{ entity.value.getType() }}</td>
+                    <td>{{ entity.value.getEntity() }}</td>
```

Comment:
what does this display? the whole entity serialized as JSON - I think take this out - the `EntityDetailsComponent` should show this

Reply:
<td>{{ entity.value.getType() }}</td> - shows 1 because these are of type entity (from the enum) probably unnecessary, I will remove it. 
                    <td>{{ entity.value.getEntity() }}</td> - shows the kind I think. I can remove this too

Reply:
On second thought - somehow, entity.value.getEntity() and not entity.value.getKind() seems to show the "kind" for the entity. So should I keep entity and remove kind instead?